docs/vagrant.md

```diff
@@ -7,25 +7,155 @@ You can use following guide to setup a cluster in a virtual machine.
 * Vagrant
 
 ### Preparation
-* bin/build-release.sh must rename _release/apache-storm-${RELEASE} to _release/storm-mesos-${RELEASE}
-* sudo echo "192.168.50.101 master" >> /etc/hosts
-* sudo echo "192.168.50.102 slave"  >> /etc/hosts
+1. Add name lookup entries for the 2 VMs:
+
+  ```shell
+  sudo sh -c 'echo "192.168.50.101 master" >> /etc/hosts'
+  sudo sh -c 'echo "192.168.50.102 slave"  >> /etc/hosts'
+  ```
+
+2. Update `storm.yaml`, specifically ensuring you set these fields:
+
+  ```shell
+  mesos.master.url: "zk://master:2181/mesos"
+  storm.zookeeper.servers:
+  - "master"
+  # Modify package version as appropriate
+  mesos.executor.uri: "file:///usr/local/storm/storm-mesos-0.9.6.tgz"
+  ```
 
 ### Run
 
 ```shell
 bin/build-release.sh
-cp vagrant/Vagrantfile .
 ```
 
 To start the cluster run following:
 ```shell
 vagrant up
 ```
-At this point the VM will have two node mesos cluster running, node 1 (master) will have a mesos master and slave running and node 2 (slave) will have only a mesos slave running. Docker has been installed as well.
+This will bring up a 2-node mesos cluster:
```

Comment:
this doesn't seem to be true.  Here's the code:
```
  if enable_second_slave
    config.vm.define "slave" do |node|
      node.vm.box = "trusty64-vm-2"
      node.vm.hostname = "slave"
      node.vm.network :private_network, ip: slave_ip
      node.vm.provision "shell", path: "vagrant/start-mesos-slave.sh", args: [master_ip, slave_ip]
    end
  end
```
and a bit earlier in the Vagrantfile we have:
```
Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
  master_ip = "192.168.50.101"
  slave_ip  = "192.168.50.102"
  # By default we only launch the 1st VM ("master") which has all the needed components:
  #  zookeeper, mesos-master, mesos-slave, storm UI, mesos-nimbus
  # However, you may want to enable the 2nd VM to better emulate a real cluster.
  enable_second_slave = false
```